0400-results.Rmd

```diff
@@ -323,6 +323,7 @@ tab_hab_summary %>%
 
 ```{r tab-wshd-sum-all}
 fpr::fpr_table_wshd_sum() %>% 
+  na_if(-507) %>% 
```

Comment:
since this number rears its head in more than one place it would kill 2 birds with one stone to do it when it comes out of tables.R.  You should comment your code so that we know why this line is there.  Not a fan of calling out a particular number from a dataframe but 

#remove negative numbers (would be better to do this for all the columns at once but I'd have to look it up to remember how the syntqax for using `dplyr::across' jives with `dplyr::case_when`
`mutate(elev_min = case_when(elev_min < 0 ~ na_real, T ~ elev_min)`